website/docs/r/is_security_group_rule.html.markdown

```diff
@@ -105,8 +105,8 @@ Review the argument references that you can specify for your resource.
 - `icmp` - (Optional, List) A nested block describes the `icmp` protocol of this security group rule.
 
   Nested scheme for `icmp`:
-  - `type`- (Required, Integer) The ICMP traffic type to allow. Valid values from 0 to 254.
-  - `code` - (Optional, Integer) The ICMP traffic code to allow. Valid values from 0 to 255.
+  - `type`- (Optional, Integer) The ICMP traffic type to allow. Valid values from 0 to 254. If absent, all codes are allowed.
+  - `code` - (Optional, Integer) The ICMP traffic code to allow. Valid values from 0 to 255. If absent, all codes are allowed.
```

Comment:
Change `If absent, all codes are allowed` to `If unspecified, all codes are allowed`